styles/images.css

```diff
@@ -0,0 +1,13 @@
+.image-res {
+  width: 100%;
```

Comment:
Could have added a better name..like img-responsive... res did not gave clear meaning